rust/src/main.rs

```diff
@@ -316,4 +317,23 @@ mod tests {
         let second = format_toml(got.as_str(), &settings);
         assert_eq!(second, got);
     }
+
+    /// Test that the column width is respected,
+    /// and that arrays are neither exploded nor collapsed without reason
+    #[rstest]
+    fn test_column_width(data: PathBuf) {
+        let start = read_to_string(data.join("column-width.start.toml")).unwrap();
+        let settings = Settings {
+            column_width: 80,
+            indent: 4,
+            keep_full_version: false,
+            max_supported_python: (3, 12),
+            min_supported_python: (3, 12),
+        };
+        let got = format_toml(start.as_str(), &settings);
+        let expected = read_to_string(data.join("column-width.expected.toml")).unwrap();
```

Comment:
Don't create separate files, inline the test like other tests do.

Reply:
Will do, but be aware that since I change the settings, this won’t result in adding a parameter to the big parametrized test, it’ll still be a separate test case.

Reply:
Yeah that is fine.